### PR TITLE
The AI's internal camera functions better

### LINKED ISSUE #28637: AI core camera chunks do not update properly in some areas
>Server revision compiled on: 2017-06-20
>The following pull requests are currently test merged:
>#28616: 'Makes movement delay calculation not round to the tick' by MrStonedOne
>Based off origin/master commit: 885c49

When unbolted and taken to the escape shuttle, the AI core has no sight in its vicinity. It can still hear and speak as normal, but also still gets messages to the effect of "Urist McUristface put a MultiPhase X07 Energy Gun into a security satchel". Which leads me to think the AI core is still receiving some sight messages, despite not actually being able to see.

I currently like to think the AI core has a super-advanced audial frequency analysis system that can tell exactly what is putting what into what, based on the sound it makes.

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #28637
As sideeffect, cyborg internal cameras will update a little more often.

### CODE REVIEW

**Comment by GunHog** [comment]
Thank you from the bottom of our metallic hearts!

**Comment by Shadowlight213** [comment]
So what was causing the bug? I'm just curious, and cant quite tell from the pr.

File: code/datums/wires/robot.dm [hunk 1]
```diff
@@ -75,10 +75,10 @@
...
 				R.lawupdate = FALSE
 		if (WIRE_CAMERA) // Disable the camera.
-			if(!isnull(R.camera) && !R.scrambledcodes)
-				R.camera.status = mend
-				R.camera.toggle_cam(usr, 0)
-				R.visible_message("[R]'s camera lense focuses loudly.", "Your camera lense focuses loudly.")
+			if(!QDELETED(R.builtInCamera) && !R.scrambledcodes)
+				R.builtInCamera.status = mend
+				R.builtInCamera.toggle_cam(usr, 0)
+				R.visible_message("[R]'s camera lens focuses loudly.", "Your camera lens focuses loudly.")
```
Comment (RemieRichards) [inline code/datums/wires/robot.dm]: You can't change this one.
Contribution guidelines.

**Review by RemieRichards — DISMISSED** [review]
Breaks guidelines

File: code/datums/wires/robot.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/datums/wires/robot.dm]: this spelling does not exist it is actively a misspelling no matter where you're from

Comment (ShizCalev) [inline code/datums/wires/robot.dm]: @RemieRichards what is it breaking?

https://en.oxforddictionaries.com/definition/lens - UK spelling
https://www.merriam-webster.com/dictionary/lens - US spelling

Lens is the singular form, lenses is the plural form, lense is a common typo that assumes there is an extra -e.

https://www.dpreview.com/forums/post/10586787
Forum post from a British photographer calling people who think lense is a British spelling silly.

Comment (ChangelingRain) [inline code/datums/wires/robot.dm]: well, okay, that's technically inaccurate. it's a very archaic spelling.

Comment (RemieRichards) [inline code/datums/wires/robot.dm]: this one's a joke though I'll admit poorly planned.
I thought joan'd get it.

File: code/datums/wires/robot.dm [hunk 2]
```diff
@@ -45,8 +45,8 @@
 					else
 						R.notify_ai(TRUE)
 		if(WIRE_CAMERA) // Pulse to disable the camera.
-			if(!isnull(R.camera) && !R.scrambledcodes)
-				R.camera.toggle_cam(usr, 0)
+			if(!QDELETED(R.builtInCamera) && !R.scrambledcodes)
+				R.builtInCamera.toggle_cam(usr, 0)
 				R.visible_message("[R]'s camera lense focuses loudly.", "Your camera lense focuses loudly.")
```
Comment (ShizCalev) [inline code/datums/wires/robot.dm]: Would you mind correcting that typo here too?

File: code/datums/wires/robot.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/datums/wires/robot.dm]: Brits confirmed for being unable to spell.

**Comment by GunHog** [comment]
> "lense is actually a misspelling of flense"

I giggled at this a little too much!

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**